jenkins-jobs/pipelines/openshift_pipeline.groovy

```diff
@@ -0,0 +1,102 @@
+pipeline {
+    agent {
+        label 'Slave'
+    }
+
+    stages {
+        stage('Checkout') {
+            when {
+                expression { !params.PRODUCT_BUILD }
+            }
+            steps {
+                checkout([
+                        $class           : 'GitSCM',
+                        branches         : [[name: "${DBZ_GIT_BRANCH}"]],
+                        userRemoteConfigs: [[url: "${DBZ_GIT_REPOSITORY}"]],
+                        extensions       : [[$class           : 'RelativeTargetDirectory',
+                                             relativeTargetDir: 'debezium']],
+                ])
+
+                checkout([
+                        $class           : 'GitSCM',
+                        branches         : [[name: "${STRZ_GIT_BRANCH}"]],
+                        userRemoteConfigs: [[url: "${STRZ_GIT_REPOSITORY}"]],
+                        extensions       : [[$class           : 'RelativeTargetDirectory',
+                                             relativeTargetDir: 'strimzi']],
+                ])
+            }
+        }
+
+        stage('Configure') {
+            steps {
+                script {
+                    env.OCP_PROJECT_DEBEZIUM="debezium-${BUILD_NUMBER}"
+                    env.OCP_PROJECT_MYSQL="debezium-${BUILD_NUMBER}-mysql"
+                    env.OCP_PROJECT_POSTGRESQL="debezium-${BUILD_NUMBER}-postgresql"
+                }
+                withCredentials([
+                        usernamePassword(credentialsId: "${OCP_CREDENTIALS}", usernameVariable: 'OCP_USERNAME', passwordVariable: 'OCP_PASSWORD'),
+                ]) {
+                    sh '''
+                    set -x            
+                    oc login ${OCP_URL} -u "${OCP_USERNAME}" --password="${OCP_PASSWORD}" --insecure-skip-tls-verify=true >/dev/null
+                    oc new-project ${OCP_PROJECT_DEBEZIUM}
+                    oc new-project ${OCP_PROJECT_MYSQL}
+                    oc new-project ${OCP_PROJECT_POSTGRESQL}
+                    '''
+                    sh '''
```

Comment:
Why is this in two different `sh` steps?

Reply:
Just that these are two distinct operations so I used separate sh steps as a way to logically divide them.